api/controllers/UserController.js

```diff
@@ -50,33 +51,37 @@ module.exports = {
       return res.badRequest(err);
     }
   },
+
   reLogin: function (req, res) {
     userAPI
       .findUserByUsername(req.body.username)
       .then(function gotUser(user) {
+        const { game: gameId, loggedIn } = req.session;
         const checkPass =
-          !req.session.loggedIn && req.body.password
+          !loggedIn && req.body.password
             ? passwordAPI.checkPass(req.body.password, user.encryptedPassword)
             : null;
-        const promiseGame = gameService.populateGame({ gameId: user.game });
+        const promiseGame = gameId ? gameService.findGame({ gameId }) : null;
```

Comment:
Oops. Swapped `.populateGame()` for `.findGame()` which doesn't grab the points. This caused the CI failure